docs/tutorials/recording.md

```diff
@@ -76,11 +59,25 @@ export CARET_IGNORE_TOPICS="/clock:/parameter_events"
 
 # Launch the target application, demos_end_to_end_sample
 ros2 launch caret_demos end_to_end_sample.launch.py
-^C # Finish with Ctrl+C after several seconds
+```
+
+### Starting recording
+
+Open a new terminal and record the performance data.
+
+```bash
+source /opt/ros/humble/setup.bash
+source ~/ros2_caret_ws/install/local_setup.bash
+
+# set a destination directory. ~/.ros/tracing is default.
+mkdir -p ~/ros2_ws/evaluate
+export ROS_TRACE_DIR=~/ros2_ws/evaluate
+
+ros2 caret record -s e2e_sample
 
-[WARNING] [launch]: user interrupted with ctrl-c (SIGINT)
-[end_to_end_sample-1] [INFO] [1631481246.160958444] [rclcpp]: signal_handler(signal_value=2)
-[INFO] [end_to_end_sample-1]: process has finished cleanly [pid 722356]
+# Start recording with pressing Enter key
+# > All process tarted recording.
+# > press enter to stop...
 ```
 
 You can finish the target application and LTTng session.
```

Comment:
> You can finish the target application and LTTng session. LTTng session will be closed if you push Enter key on the terminal where the LTTng session runs.

This line can be removed because a user doesn't need to finish the target application and operation (press Enter key) is described in the above command snippet.

Reply:
addressed [address comments except for design pages](https://github.com/tier4/CARET_doc/pull/118/commits/13f63419058e3a7ffc2da8ae2ff58ebbbc1b7c63)

Reply:
@takeshi-iwanari 
I modified pages except for design.
Could you reveiw this again?

Reply:
LGTM